From d8ec4d79b31c90763577500a3f853587cba50774 Mon Sep 17 00:00:00 2001
From: Daire McNamara <daire.mcnamara@microchip.com>
Date: Wed, 2 Sep 2020 13:48:09 +0100
Subject: [PATCH v1 1/1] cpufreq: Add support for Microchip PolarFire to
 cpufreq-dt-platdev

<Signed-off-by: Daire McNamara daire.mcnamara@microchip.com>
---
 arch/riscv/Kconfig                   | 8 ++++++++
 drivers/cpufreq/cpufreq-dt-platdev.c | 1 +
 2 files changed, 9 insertions(+)

diff --git a/arch/riscv/Kconfig b/arch/riscv/Kconfig
index e9e2c1f0a690..dd6991467df1 100644
--- a/arch/riscv/Kconfig
+++ b/arch/riscv/Kconfig
@@ -347,6 +347,14 @@ config RISCV_SBI_V01
 	  deprecated in future once legacy M-mode software are no longer in use.
 endmenu
 
+menu "CPU Power Management"
+
+source "drivers/cpufreq/Kconfig"
+
+source "drivers/cpuidle/Kconfig"
+
+endmenu
+
 menu "Boot options"
 
 config CMDLINE
diff --git a/drivers/cpufreq/cpufreq-dt-platdev.c b/drivers/cpufreq/cpufreq-dt-platdev.c
index bd2db0188cbb..c3251e050e16 100644
--- a/drivers/cpufreq/cpufreq-dt-platdev.c
+++ b/drivers/cpufreq/cpufreq-dt-platdev.c
@@ -16,6 +16,7 @@
  * platforms using "operating-points" (V1) property.
  */
 static const struct of_device_id whitelist[] __initconst = {
+	{ .compatible = "microchip,mpfs-dev", },
 	{ .compatible = "allwinner,sun4i-a10", },
 	{ .compatible = "allwinner,sun5i-a10s", },
 	{ .compatible = "allwinner,sun5i-a13", },
-- 
2.25.1